### PR TITLE
xfce.thunar-media-tags-plugin: init at 0.3.0

### PR DESCRIPTION
**Pull request by ncfavier**
cc @romildo 

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested that this actually works
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/xfce/thunar-plugins/media-tags/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    taglib
+  ];
+
+  preConfigure = ''
+    ./autogen.sh
+  '';
+
+  meta = with lib; {
+    description = "Thunar plugin providing tagging and renaming features for media files";
+    maintainers = with maintainers; [ ncfavier ];
```
Comment (SuperSandro2000) [inline pkgs/desktops/xfce/thunar-plugins/media-tags/default.nix:38]: Missing license and homepage.

File: pkgs/desktops/xfce/thunar-plugins/media-tags/default.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    thunar
+    exo
+    gtk3
+    libxfce4util
+    taglib
+  ];
+
+  preConfigure = ''
+    ./autogen.sh
+  '';
```
Comment (SuperSandro2000) [inline pkgs/desktops/xfce/thunar-plugins/media-tags/default.nix:34]: This could be replaced with autoreconfHook.

Comment (ncfavier) [reply pkgs/desktops/xfce/thunar-plugins/media-tags/default.nix:34]: Right, it seems to just work if I remove the `preConfigure` hook, do you know why? Should I still include `autoreconfHook` explicitly?

File: pkgs/desktops/xfce/thunar-plugins/media-tags/default.nix (hunk 1 above)
Comment (ncfavier) [reply pkgs/desktops/xfce/thunar-plugins/media-tags/default.nix:38]: Those are set by `mkXfceDerivation`.

File: pkgs/desktops/xfce/thunar-plugins/media-tags/default.nix (hunk 2 above)
Comment (ncfavier) [reply pkgs/desktops/xfce/thunar-plugins/media-tags/default.nix:34]: Ok so `mkXfceDerivation` pulls `xfce4-dev-tools` which pulls `autoreconfHook`.

**Comment by SuperSandro2000** [comment]
can you ping me in a couple of days if no one merged this?

**Comment by ncfavier** [comment]
Sure

**Comment by ncfavier** [comment]
@SuperSandro2000 I think we can merge this now

**Pull request merged**